src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingContext.cs

```diff
@@ -239,6 +241,8 @@ public async Task<FormattingContext> WithTextAsync(SourceText changedText)
         [Conditional("DEBUG")]
         private static void ValidateComponents(RazorCodeDocument oldCodeDocument, RazorCodeDocument newCodeDocument)
         {
+            if (SkipValidateComponents) return;
```

Comment:
Personally I don't like braceless if-statements. Firstly it breaks the normal read pattern because the statement is on the right instead of below, and secondly it makes any needed refactoring's noisy git-wise.